SimpleWAWS/Code/SiteNameGenerator.cs

```diff
@@ -10,5 +10,10 @@ public static string GenerateName()
             //first 4 parts of a guid
             return string.Format(CultureInfo.InvariantCulture, "{0}-0ee0-4-231-b9ee", Guid.NewGuid().ToString().Substring(0, 8));
         }
+        public static string GenerateVmName()
+        {
+            //first 4 parts of a guid
+            return string.Format(CultureInfo.InvariantCulture, "a{0}-0ee0-4-231-b9ee", Guid.NewGuid().ToString().Substring(0, 7));
```

Comment:
FYI - VMs probably don't have to adhere to the naming convention we have, but it's up to you, it's as good as any name, but maybe using an actual guid would prevent any possibility of collisions.